hijack/locale/fr/LC_MESSAGES/django.po

```diff
@@ -13,30 +13,33 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
 "X-Language: fr\n"
 "X-Source-Language: en\n"
-"X-Generator: Poedit 2.4.2\n"
+"X-Generator: Poedit 3.4.2\n"
 
-#: contrib/admin/admin.py:35
-msgid "hijack user"
-msgstr "détourner un utilisateur"
+#: contrib/admin/admin.py:64
+msgid "impersonate user"
+msgstr "usurper l'identité d'un utilisateur"
 
-#: contrib/admin/templates/hijack/contrib/admin/button.html:8
-msgid "hijack"
-msgstr "détourner"
+#: contrib/admin/templates/hijack/contrib/admin/button.html:7
+msgctxt "verb"
+msgid "impersonate"
+msgstr "se faire passer pour quelqu'un d'autre"
```

Comment:
Way too long and inconsistent with others. Just use *détourner*

Reply:
```suggestion
msgstr "détourner"
```